src/seedu/addressbook/data/tag/Tagging.java

```diff
@@ -0,0 +1,30 @@
+package seedu.addressbook.data.tag;
+
+import seedu.addressbook.data.person.Person;
+
+public class Tagging {
+	
+	private static final String PRINT_FORMAT = "%s %s %s";
+	private static final String ADDED_SIGN = "+";
+	private static final String DELETED_SIGN = "-";
+	
+	private final Person person;
+	private final Tag tag;
+	private final boolean isAdded;
+	
+	public Tagging(Person person, Tag tag, boolean isAdded){
```

Comment:
A boolean variable works here, but an `enum` may be more appropriate because the two values we need are `added` and `deleted`, not `true` and `false`.